src/FactorGraph01.jl

```diff
@@ -351,6 +351,43 @@ function getOutNeighbors(dfg::T, vertSym::Symbol; needdata::Bool=false, ready::I
   return nodes
 end
 
+
+
+function DefaultNodeDataParametric(dodims::Int,
```

Comment:
bread crumb for future consolidation with existing `setDefaultNodeData`

Reply:
That was the plan. For now I tried to make the code as detached as possible to ease merging at a later stage. I suggest we carry on in this way until it is merged. We can then do smaller branches to do consolidation.